### DISCUSSION TITLE
SwayOS for Arch Linux!

### DISCUSSION THREAD

**Discussion by rik1599** [swayos/swayos.github.io#34] [Show and tell]
Instructions [here](https://github.com/rik1599/swayos/tree/arch).
Same aspect and software than Debian version with some little differences:

- ~Default shell remains bash (zsh not installed)~
- Gnome software not installed (useless with Arch packages, terminal is better)
- Yay AUR helper pre-installed
- Natural scrolling disabled 
- added Win+L shortcut to lock the computer
- tofi instead of wofi as launcher
- wcp and wfl already pre-installed in the default configuration (necessary for compatibility issues)
- system-config-printer removed (uses CUPS web interface)

Any suggestion or contribution is appreciated.

UPDATE: zsh installed with [Oh My Zsh](https://ohmyz.sh/) framework

**Comment by BaaridunNasr** [comment]
Instructions link seems to be brokem
